### ISSUE TITLE
SF4 : is SensioFrameworkExtraBundle disabling autowiring ?

### ISSUE THREAD

**Issue by CharlyPoppins** [sensiolabs/SensioFrameworkExtraBundle#611]
Hi,

I need **SensioFrameworkExtraBundle** to use **@Security** annotation.

Then I've got a strange behaviour :

Enabling `Sensio\Bundle\FrameworkExtraBundle\SensioFrameworkExtraBundle::class` in `bundles.php` triggger this error :
```
Argument 1 passed to App\Controller\PanierController::selectionSuccess() must be an instance of CleanCms\PageBundle\Manager\PageManager, null given, called in /Users/charly/Documents/Sources/Sites/sculptures-bidal/vendor/symfony/http-kernel/HttpKernel.php on line 150
```
If I remove `Sensio\Bundle\FrameworkExtraBundle\SensioFrameworkExtraBundle::class` from 'bundles.php` all is working good. But it feels weird to require a library and to use it (**@Security**) without enabling it.

**Comment by danielkay** [comment]
We are also having this issue - weirdly enough, however, the problem goes away if we fix our dependency to v5.2.4, and looking at the date of the 5.3.0 release and the timing of this issue I would presume this is the version on which you were finding this problem to exhibit itself (unless you were building against dev-master of SensioFrameworkExtraBundle, which I presume you weren't!)

**Comment by CharlyPoppins** [comment]
Hello, nearly one year after I'm giving a second try then went to the same error.

I found out that disabling converter solve the issue : ```sensio_framework_extra:
    request:     { converters: false}```

The real problem was a @ParamConverter with method **supports** returned always **true** then method **apply** returning always **null** -> that's why services autowired were null.
